tests/test_files.py

```diff
@@ -32,5 +31,50 @@ def test_files_full(self):
         parse_file_full('tests/recs/de-13.07.aoe2record')
 
     def test_files_fast(self):
-        for path in glob.glob('tests/recs/*'):
```

Comment:
Let's do this:

```
skip = {"tests/recs/de-50.6-scenario.aoe2record", ...}
for path in glob.glob('tests/recs/*'):
    if path in skip:
        continue
    parse_file_fast(path)
```

That way we don't need to update this test every time we add another test file.